pymtl3/passes/backends/verilog/tbgen/test/VerilogTBGen_test.py

```diff
@@ -0,0 +1,104 @@
+#=========================================================================
+# ImportedObject_test.py
+#=========================================================================
+# Author : Peitian Pan
+# Date   : Jun 2, 2019
+"""Test if the imported object works correctly."""
+
+from os.path import dirname
+
+import pytest
+
+from pymtl3 import SimulationPass
+from pymtl3.datatypes import Bits1, Bits32, Bits48, Bits64, clog2, mk_bits
+from pymtl3.dsl import Component, InPort, Interface, OutPort, Placeholder, connect
+from pymtl3.passes.backends.verilog import (
+    TranslationConfigs,
+    TranslationImportPass,
+    VerilatorImportConfigs,
+    VerilogPlaceholderConfigs,
+    VerilogPlaceholderPass,
+    VerilogTBGenPass,
+)
+from pymtl3.passes.rtlir.util.test_utility import do_test
+from pymtl3.stdlib.test import TestVectorSimulator
+
+from ...testcases import CaseConnectArrayBits32FooIfcComp
+
+
+def local_do_test( _m ):
+  _m.elaborate()
+  _m.apply( VerilogPlaceholderPass() )
+  m = TranslationImportPass()( _m )
+  m.verilog_tbgen = True
+  m.apply( VerilogTBGenPass() )
+  sim = TestVectorSimulator( m, _m._test_vectors, _m._tv_in, _m._tv_out )
+  sim.run_test()
+
+def test_normal_queue( do_test ):
```

Comment:
To be clear, the tests here just generate the testbench but do not simulate it?

Reply:
i assume no verilog simulator ...